.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        arch:
+          - amd64
+          - arm64
+        os:
+          - linux
+        include:
+          - arch: amd64
+            os: darwin
+          - arch: amd64
+            os: windows
+    runs-on: ubuntu-20.04
```

Comment:
We should use a Debian base image if we can because that's what the container releases use.

Reply:
This is the list of available base images: https://github.com/actions/virtual-environments#available-environments
Note that this is not a Docker container, this is a full-blown virtual machine.

What we can do is to perform build inside docker container. Stay tuned, I'm updating this PR.

Reply:
Done, it uses `xgo` now.